javascript/detail_and_listview.js

```diff
@@ -157,10 +174,13 @@ var createRouteCardSkeleton = function(object, walkerLoc) {
     divReviewStars.setAttribute("class", "review-stars");
     divReviews.appendChild(divReviewStars);
 
-    getAmountOfStars(divReviewStars);
+    getAmountOfStars(divReviewStars, id);
```

Comment:
I don't understand what ID is and how it is different then object?

Reply:
I don't know what object was intended to be but when I tried to use it, it was giving me [object object]

Reply:
The id itself just represents the actual id string value